.gitignore

```diff
@@ -5,3 +5,4 @@ repl-temp-*
 lib/index.js
 node_modules/
 tests/**/elm-package.json
+!tests/subdirectory/elm-package.json
```

Comment:
We need to keep `elm-packge.json` for this test because it has to have a custom value for `"source-directories"`